src/components-examples/material/table/table-column-styling/table-column-styling-example.html

```diff
@@ -0,0 +1,29 @@
+<table mat-table [dataSource]="dataSource" class="mat-elevation-z8">
+  <!-- Position Column -->
+  <ng-container matColumnDef="position">
```

Comment:
Should we prefix these names e.g. `demo-position`, `demo-name`? otherwise classes like `.mat-column-position`, `.mat-column-name` sound like they could just be generic things